assets/hub-search-bar.js

```diff
@@ -0,0 +1,32 @@
+docsearch({
+  apiKey: "e3b73ac141dff0b0fd27bdae9055bc73",
+  indexName: "pytorch",
+  inputSelector: "#hub-search-input",
+  debug: true // Set debug to true if you want to inspect the dropdown
+});
+
+$("#hub-search-icon").on("click", function() {
+  $(this).hide();
+  $("#hub-close-search").show(200);
+  $("#hub-divider")
+    .addClass("active-hub-divider")
+    .show(200);
+  $("#hub-search-input")
+    .show()
+    .focus();
+  $("#dropdownFilter").hide();
+});
+  
+$("#hub-close-search").on("click", function() {
+  $(this).hide();
+  $("#hub-search-icon").show(200);
+  $("#hub-search-input").hide(200);
+  $("#hub-divider")
+    .attr("style", "")
+    .removeClass("active-hub-search")
+    .removeClass("active-hub-divider");
+  $("#hub-search-input")
+    .removeClass("active-search-icon")
+    .val("");
+  $("#dropdownFilter").fadeIn("slow");
+});
```

Comment:
You should double-check your editor preferences and make sure that a final empty line is added when you save a file.